_docs_operate/configuration.md

```diff
@@ -103,7 +103,7 @@ The Connector provides the following configuration parameters:
 
 You can validate the config using our [schema file](https://raw.githubusercontent.com/nmshd/cns-connector/main/config.schema.json). This is possible for example with [VSCode](https://code.visualstudio.com/docs/languages/json#_json-schemas-and-settings) or online tools like [jsonschemavalidator.net](https://www.jsonschemavalidator.net).
 
-### debug `availbable since version 3.3.0` {#debug}
+### debug `available since version 3.3.0` {#debug}
```

Comment:
This is the kind of fix we discussed that we want to perform in separate PRs and not within a PR that is dedicated to a different topic. I don't mind it here too much, since I'm already done reviewing this PR, but please keep it in mind for the future.

Reply:
I must confess that I did this with full awareness of what was being discussed :D You are absolutely right, in any case, it should be in mind of every developer that we should stick to the subject within a PR. However, as I understand it from our discussion and as I think, in some cases with simple PRs it should be okay to fix isolated typos.